### PR TITLE
doc: auto-tls example in security guide

### PR DESCRIPTION
**Pull request by heyitsanthony**

### CODE REVIEW

File: Documentation/op-guide/security.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ etcd takes several certificate related configuration options, either through com
 
 `--trusted-ca-file=<path>`: Trusted certificate authority.
 
+`--auto-tls`: Use self-signed certificates for connections with clients.
```
Comment (xiang90) [inline Documentation/op-guide/security.md]: Use auto generated self-signed certificates for connections with clients.

File: Documentation/op-guide/security.md [hunk 2]
```diff
@@ -32,6 +34,8 @@ The peer options work the same way as the client-to-server options:
 
 `--peer-trusted-ca-file=<path>`: Trusted certificate authority.
 
+`--peer-auto-tls`: Generate and use self-signed certificates for SSL/TLS connections between peers.
```
Comment (xiang90) [inline Documentation/op-guide/security.md]: remove SSL just say TLS?

Comment (xiang90) [inline Documentation/op-guide/security.md]: Use auto generated and self-signed certificates for TLS connections between peers.

**Comment by xiang90** [comment]
LGTM.

**Comment by gyuho** [comment]
LGTM. Thanks!

**Pull request merged**